README.md

```diff
@@ -117,6 +117,14 @@ cd test_runner
 pytest
 ```
 
+If you see a `Found interfering processes running` error, stop pageserver and postgres nodes for all branches:
+
+```sh
+./target/debug/zenith pg stop migration_check
+./target/debug/zenith pg stop main
+./target/debug/zenith stop
```

Comment:
That wouldn't help with system-wide postgres. I'd rather explain what processes we are talking about in error message. Something like `Found interfering processes running (that is 'postgres' or 'wal_acceptor' or 'pageserver')`.

I hope we may ditch that check at all after https://github.com/zenithdb/zenith/pull/353 will be merged

Reply:
Also `migration_check` is test specific branch name.

Reply:
That's true. I believe the error message is already clear enough.

Here I wanted to streamline the "opened README and want to check that everything is running without going into details" situation. Here is another take, PTAL.